backend/services/applications.js

```diff
@@ -178,7 +178,16 @@ async function createApplication(raw_application) {
  * Returns a list of JSON objects representing all applications in the system.
  * Can be filtered using options (e.g. to fetch only open applications, etc.)
  */
-async function getAllApplications(start_date, end_date) {
+async function getAllApplications(start_date, end_date, completed = null) {
+  if (completed != null) {
+    return Application.find({
+      completed: false,
```

Comment:
Instead of null, I think it makes more sense to make it true or false. So maybe the default can be final = false, meaning we don't only want the final applications.

Reply:
Not sure if you planned on changing this, or do you think it is better the way it is?

Reply:
I didn't resolve this but yeah I already changed this.